### PR TITLE
Hesa clean doc

### PR DESCRIPTION
**Pull request by hesa**
After review of the README and EXAMPLES file I propose some updates - as in the commits

### CODE REVIEW

File: EXAMPLES.md [hunk 1]
```diff
@@ -113,20 +113,30 @@ $ flict list
 
 # Create a policy report of the Europe project
 
-***This feature is currently unavailable***
+Create a compatibility, `fontconfig-report.json`, report for `fontconfig.json`:
+```
```
Comment (priv-kweihmann) [inline EXAMPLES.md]: Minor nitpick - between the text and a unformatted box there should always be a blank line.
Additionally pls specify the content of the code box by e.g. ```shell or similar

File: README.md [hunk 2]
```diff
@@ -96,23 +96,33 @@ flict outputs a report as well as an exit code.
...
 
 ### JSON
 
-This is currently rewritten and not available.
+Default. Available for all commands.
 
 ### Markdown
 
-Using this format you can create txt, html, pdf and what format pandoc can create from markdown.
+Using this format you can create txt, html, pdf and what format pandoc can create from markdown. Partially supported.
```
Comment (priv-kweihmann) [inline README.md]: I would rephrase this here, just pointing to pandoc (and its supported outputs)

File: EXAMPLES.md (hunk 1 above)
Comment (hesa) [reply EXAMPLES.md]: I shell fix that

**Review by priv-kweihmann — APPROVED** [review]
